package-lock.json

```diff
@@ -9,48 +9,42 @@
       "version": "1.0.0",
       "license": "MIT",
       "dependencies": {
-        "@kontent-ai/delivery-sdk": "^12.1.0",
-        "@kontent-ai/eslint-config": "^0.1.3",
-        "@kontent-ai/webhook-helper": "^2.0.1",
-        "@netlify/functions": "^1.2.0",
-        "@testing-library/jest-dom": "^5.16.5",
-        "@testing-library/react": "^13.3.0",
-        "@testing-library/user-event": "^13.5.0",
-        "@types/jest": "^27.5.2",
-        "@types/node": "^16.11.56",
-        "@types/react": "^18.0.17",
-        "@types/react-dom": "^18.0.6",
-        "axios": "^0.27.2",
-        "dprint": "^0.36.1",
-        "eslint": "^8.41.0",
+        "@kontent-ai/delivery-sdk": "^14.8.0",
+        "@kontent-ai/webhook-helper": "^2.1.0",
```

Comment:
new webhooks should be realeased soon. let's see if dependabot will announce it

Reply:
Dependabot only announces fixes for security issues. I don't believe that this update will be marked as such. 😄
But I updated the package anyway.